packit_service/worker/handlers/copr.py

```diff
@@ -319,8 +319,15 @@ def run(self):
 
         # https://pagure.io/copr/copr/blob/master/f/common/copr_common/enums.py#_42
         if self.copr_event.status != COPR_API_SUCC_STATE:
-            failed_msg = "RPMs failed to be built."
-            packit_dashboard_url = get_copr_build_info_url(self.build.id)
+            if (
+                srpm_build := self.build.get_srpm_build()
+            ) and srpm_build.status == BuildStatus.failure:
```

Comment:
if this is `True`, hasn't the message and URL been reported to the user previously?

Reply:
sorry for not elaborating more, I meant whether we cannot just log it and return here, so that the status is not updated another time

Reply:
Good point, let me change it.